src/main/java/meteordevelopment/meteorclient/systems/modules/player/FakePlayer.java

```diff
@@ -42,6 +42,14 @@ public class FakePlayer extends Module {
         .build()
     );
 
+    public final Setting<Boolean> allowDamage = sgGeneral.add(new BoolSetting.Builder()
+        .name("Allow Damage")
```

Comment:
change to `allow-damage` for consistency with the rest of the entire project's settings

Reply:
Forgot to change this when testing, thanks.